src/main/java/com/github/danielflower/mavenplugins/gitlog/GenerateMojo.java

```diff
@@ -187,6 +199,18 @@
 	 */
 	@Parameter
 	private String path;
+	
+	/**
+	 * Include in the changelog the commits after the last release
+	 */
+	@Parameter(defaultValue = "false")
+	private boolean includeOnlyCommitsAfterRelease;
+	
+	/**
+	 * the tag name of release

```

Comment:
the option includeOnlyCommitsAfterRelease allows to generate only the commits of the last release.
If **includeOnlyCommitsAfterRelease** is true the generated report contains only the commits after the last delivery.
If option X is false the generated report contains all the commits of the branch.
Example configuration :
![maven-gitlog-pluginpom xml at master edahenitmaven-gitlog-plugin - google chrome](https://cloud.githubusercontent.com/assets/15674407/23356376/dcac08b4-fcd9-11e6-8bff-2b56d1a34aa0.jpg)

This diagram explains the case where the option **includeOnlyCommitsAfterRelease**  is true :
![microsoft powerpoint - gitlog pptm](https://cloud.githubusercontent.com/assets/15674407/23356763/5e511e62-fcdb-11e6-90ca-1070a031b1cd.jpg)